eslint.config.mjs

```diff
@@ -0,0 +1,16 @@
+import globals from 'globals';
```

Comment:
```suggestion
/**
 * @file Dynamic Babel configuration.
 *
 * Notes:
 * 1. Plugins run before presets.
 * 2. Plugin ordering is first to last.
 * 3. Preset ordering is reversed (last to first).
 *
 * @author The OpenINF Authors & Friends
 * @module {ES6Module} /eslint.config
 * @license MIT OR Apache-2.0 OR BlueOak-1.0.0
 * @see https://eslint.org/docs/latest/use/configure/plugins
 */

// -----------------------------------------------------------------------------
// Requirements
// -----------------------------------------------------------------------------

import globals from 'globals';
```

Reply:
looks like we can get rid of the stuff below (not sure why it got added automatically)